Common/Sources/Extensions/HyperwalletFee.swift

```diff
@@ -108,28 +129,75 @@ public extension HyperwalletFee {
             let min = percentFee.minimum
             let max = percentFee.maximum
             let currency = flatFee.currency
-
-            if let min = min, let max = max, let flatValue = flatValue,
-                let percentValue = percentValue, let currencySymbol = currencySymbol(currency: currency) {
-                feeFormat = "fee_mix_formatter".localized()
-                description = String(
-                    format: feeFormat, currencySymbol, flatValue, percentValue, min, max)
-            } else if let min = min, max == nil, let flatValue = flatValue,
-                let percentValue = percentValue, let currencySymbol = currencySymbol(currency: currency) {
-                feeFormat = "fee_mix_only_min_formatter".localized()
-                description = String(
-                    format: feeFormat, currencySymbol, flatValue, percentValue, min)
-            } else if min == nil, let max = max, let flatValue = flatValue,
-                let percentValue = percentValue, let currencySymbol = currencySymbol(currency: currency) {
-                feeFormat = "fee_mix_only_max_formatter".localized()
-                description = String(
-                    format: feeFormat, currencySymbol, flatValue, percentValue, max)
+            
+            if flatValue?.formatAmountToDouble() == 0 && percentValue?.formatAmountToDouble() == 0 {
+                return noFeeDescription()
+            }
```

Comment:
you can simplify the entire logic by adding else conditions to this if like 
`if flatValue?.formatAmountToDouble() == 0 && percentValue?.formatAmountToDouble() == 0 {
                return noFeeDescription()
            } else if flatValue?.formatAmountToDouble() == 0 {
                return percentFeeDescription(percentFee)
            } else if percentValue?.formatAmountToDouble() == 0 {
                return flatFeeDescription(flatFee)
            }` and then keep the existing logic of

`if let min = min, let max = max, let flatValue = flatValue,
                let percentValue = percentValue, let currencySymbol = currencySymbol(currency: currency) {
                feeFormat = "fee_mix_formatter".localized()
                description = String(
                    format: feeFormat, currencySymbol, flatValue, percentValue, min, max)
            } else if let min = min, max == nil, let flatValue = flatValue,
                let percentValue = percentValue, let currencySymbol = currencySymbol(currency: currency) {
                feeFormat = "fee_mix_only_min_formatter".localized()
                description = String(
                    format: feeFormat, currencySymbol, flatValue, percentValue, min)
            } else if min == nil, let max = max, let flatValue = flatValue,
                let percentValue = percentValue, let currencySymbol = currencySymbol(currency: currency) {
                feeFormat = "fee_mix_only_max_formatter".localized()
                description = String(
                    format: feeFormat, currencySymbol, flatValue, percentValue, max) else {...}`

Reply:
Yes that is great.